aiida/cmdline/params/options/commands/setup.py

```diff
@@ -61,12 +66,93 @@ def get_repository_path_default(ctx):
+    :return: the password
+    """
+    return get_repository_path_default(ctx)
+
+
+def get_quicksetup_database_name(ctx, param, value):  # pylint: disable=unused-argument
+    """Determine the database name to be used as default for the Postgres connection in `verdi quicksetup`
+
+    If a value is explicitly passed, that value is returned unchanged.
+
+    If no value is passed, the name will be <profile_name>_<os_user>_<hash>, where <os_user> is the name of the current
+    operating system user and <hash> is a hash of the path of the configuration directory.
+
+    Note: This ensures that profiles named ``test_...`` will have databases named ``test_...`` .
+
+    :param ctx: click context which should contain the contextual parameters
+    :return: the database name
+    """
+    if value is not None:
+        return value
+
+    config = get_config()
+    profile = ctx.params['profile'].name
+    config_hash = hashlib.md5(config.dirpath.encode('utf-8')).hexdigest()
+    database_name = '{profile}_{user}_{hash}'.format(profile=profile, user=getpass.getuser(), hash=config_hash)
+
+    return database_name
+
+
+def get_quicksetup_username(ctx, param, value):  # pylint: disable=unused-argument
+    """Determine the username to be used as default for the Postgres connection in `verdi quicksetup`
+
+    If a value is explicitly passed, that value is returned. If there is no value, the name will be based on the
+    name of the current operating system user and the hash of the path of the configuration directory.
+
+    :param ctx: click context which should contain the contextual parameters
+    :return: the username
+    """
+    if value is not None:
+        return value
+
+    config = get_config()
+    config_hash = hashlib.md5(config.dirpath.encode('utf-8')).hexdigest()
+    username = 'aiida_qs_{user}_{hash}'.format(user=getpass.getuser(), hash=config_hash)
+
+    return username
+
+
+def get_quicksetup_password(ctx, param, value):  # pylint: disable=unused-argument
+    """Determine the password to be used as default for the Postgres connection in `verdi quicksetup`
+
+    If a value is explicitly passed, that value is returned. If there is no value, the current username in the context
+    will be scanned for in currently existing profiles. If it does, the corresponding password will be used. If no such
+    user already exists, a random password will be generated.
+
+    :param ctx: click context which should contain the contextual parameters
+    :return: the password
+    """
+    from aiida.common.hashing import get_random_string
+
+    if value is not None:
+        return value
+
+    username = ctx.params['db_username']
+    config = get_config()
+
+    for available_profile in config.profiles:
+        if available_profile.database_username == username:
+            value = available_profile.database_password
+            echo.echo_info('using found password for existing user `{}`'.format(username))
```

Comment:
this now also prints appears in auto completion.
perhaps we simply get rid of it?